archive/p/pascal/README.md

```diff
@@ -6,6 +6,8 @@ Welcome to Sample Programs in Pascal!
 
 - [Fizz Buzz in Pascal][1]
 - [Hello World in Pascal][2]
+-[Baklava in Pascal][3]
```

Comment:
The link `[3]` should match the appropriate reference at the bottom of the file.
You made need to add a new issue for it.